packages/connectivity_plus_web/pubspec.yaml

```diff
@@ -24,10 +24,10 @@ dev_dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
-  integration_test: ^1.0.2
-  mockito: ^4.1.1
-  pedantic: ^1.9.2
+  integration_test: ^1.0.2+2
```

Comment:
is not migrated to null safe so to pass tests I added to the top of the files `// @dart=2.9`

Reply:
aha, yes, also I noticed that the web package structure was a bit off with the battery_plus plugin and that the integration tests were not in the right folder.

Integration tests should be detached from the plugin project, like in an "example" subproject, so we don't have a direct dependency to integration_test.

Reply:
I see. I moved the folder to the example directory